api/http/indigo_service/jsonapi.py

```diff
@@ -581,3 +582,27 @@ def make_descriptor_response(
 SourceTargetsRequest = Union[
     CommonBitsRequest, SimilaritiesRequest, MatchRequest
 ]
+
+
+# Render
+
+
+rendering_formats = {
+    "image/png": "png",
+    "image/png;base64": "png",
+    "image/svg+xml": "svg",
+    "application/pdf": "pdf",
+}
```

Comment:
We probably should add `image/svg+xml;base64` as well.

Reply:
And maybe also some JSON format with returning base64-encoded string. @khyurri, what do you think, could that be useful?